src/main/java/seedu/linkedout/logic/parser/ArgumentTokenizer.java

```diff
@@ -124,6 +129,148 @@ private static String extractArgumentValue(String argsString,
+        while (hasNextWhiteSpace(stringBeginWithPrefix) && nextSlashPrefix > whitespaceIndex) {
+            stringBeginWithPrefix = removesStringBeforeWhiteSpace(stringBeginWithPrefix, whitespaceIndex);
+            whitespaceIndex = stringBeginWithPrefix.indexOf(" "); //update white space index
+            nextSlashPrefix = slashIndex(stringBeginWithPrefix); //update slash index
+        }
+
+        return stringBeginWithPrefix;
+    }
+
+    /**
+     * removes keyword with slash
+     * require additional checking on keyword with slash e.g. java/python
+     * @param str
+     * @return string after the keyword
+     */
+    private static String removeLeadingKeywordWithSlash(String str) {
+        String keyword = str.trim();
+        int whitespaceIndex = str.indexOf(" ");
+        int keywordSlashIndex = str.indexOf("/");
+
+        //Extra check for keyword with "/" e.g.baker/chef j/engineer -> j/engineer
+        if (hasNextWhiteSpace(keyword) && whitespaceIndex > keywordSlashIndex) {
+            keyword = removesStringBeforeWhiteSpace(keyword, whitespaceIndex);
+        }
+        return keyword;
+    }
+
+    /**
+     * Removes string before whitespace
+     * @param str
+     * @param whiteSpaceIndex
+     */
+    private static String removesStringBeforeWhiteSpace(String str, int whiteSpaceIndex) {
+        return str.substring(whiteSpaceIndex).trim();
+    }
+    /**
+     * Checks if the string still has whitespace
+     * @param inputString
+     * @return True if string contains whitespace
+     */
+    private static boolean hasNextWhiteSpace(String inputString) {
+        int whitespaceIndex = inputString.indexOf(" "); // check if still has next whitespace
+        if (whitespaceIndex == -1) { // no more prefix
+            return false;
+        } else { // still has prefix
+            return true;
+        }
+    }
+
+    /**
+     * Checks if the string still has slash
+     * @param inputString
+     * @return True if string contains slash
+     */
+    private static boolean hasNextSlash(String inputString) {
+        int slashIndex = slashIndex(inputString); // check if still has next slash
+        if (slashIndex == -1) { // no more prefix
+            return false;
+        } else { // still has prefix
+            return true;
+        }
+    }
+
+    private static int slashIndex(String str) {
+        return str.indexOf("/");
+    }
+
+    /**
+     * Process for checking if input string contains any invalid prefix
+     * @param argsString
+     * @param argumentMultimap
+     * @return false if the input string contains invalid prefix
+     */
+    private static boolean checksForValidPrefix(String argsString, ArgumentMultimap argumentMultimap) {
+        boolean hasNextWhiteSpace = hasNextWhiteSpace(argsString);
+        boolean hasSlashInString = hasNextSlash(argsString);
+        boolean isInvalidPrefix = false;
+        int slashIndex;
+
+        while (hasSlashInString && hasNextWhiteSpace) {
```

Comment:
This might be a `cond1` or `cond1 || cond2` as I believe it misses out some cases upon testing on my local machine.